src/fw/flash_region/flash_region_gd25lq255e.h

```diff
@@ -29,12 +29,13 @@
 
 #define FLASH_REGION_DEF(MACRO, arg)                                                      \
   MACRO(FIRMWARE_SCRATCH, 0x0100000 /*  1024k */, arg)        /*      0x0 - 0x0100000 */  \
-  MACRO(SYSTEM_RESOURCES_BANK_0, 0x0080000 /*   512K */, arg) /* 0x0100000 - 0x0180000 */ \
-  MACRO(SYSTEM_RESOURCES_BANK_1, 0x0080000 /*   512K */, arg) /* 0x0180000 - 0x0200000 */ \
-  MACRO(SAFE_FIRMWARE, 0x0080000 /*   512k */, arg)           /* 0x0200000 - 0x0280000 */ \
-  MACRO(DEBUG_DB, 0x0020000 /*   128k */, arg)                /* 0x0280000 - 0x02A0000 */ \
-  MACRO(FILESYSTEM, 0x1D50000 /* 30016k */, arg)              /* 0x02A0000 - 0x1FF0000 */ \
-  MACRO(RSVD, 0x000E000 /*    56k */, arg)                    /* 0x1FF0000 - 0x1FFE000 */ \
+  MACRO(SYSTEM_RESOURCES_BANK_0, 0x0100000 /*   1024K */, arg) /* 0x0100000 - 0x0200000 */ \
+  MACRO(SYSTEM_RESOURCES_BANK_1, 0x0100000 /*   1024K */, arg) /* 0x0200000 - 0x0300000 */ \
```

Comment:
Do we want to go even as high as 2MB, since we seem to have a whole universe of flash available?  Then we could have more pack-in apps.

Reply:
Up to you guys, we have plenty of flash to do so.

Reply:
Another question, wouldn't it be better to move PRF after fw scratch area? This way we could always resize resources provided we backup first PFS.

Reply:
let's do 2MB for now, flash is cheap and we have way more than we ever did, and we do have to commit early to the layout

Reply:
yup, I'm OK with rearranging PRF to be after FW scratch area